fuzzing/tools/launcher.py

```diff
@@ -0,0 +1,26 @@
+# Lint as: python3
+"""
+This is the launcher script to provide a uniform command line interface 
+behind a number of arbitrary fuzzing engines.
+"""
+
+from absl import app
+from absl import flags
+import os
+
+FLAGS = flags.FLAGS
+
+flags.DEFINE_integer('timeout', 20, 'test timeout.', lower_bound=0)
```

Comment:
Call this flag `timeout_secs`, to clarify the unit of measure on the command line?